types/prismjs/components.d.ts

```diff
@@ -0,0 +1,4 @@
+export const core: Record<string, any>;
+export const languages: Record<string, any>;
+export const plugins: Record<string, any>;
+export const themes: Record<string, any>;
```

Comment:
Is this export style correct? 

I don't know how TS translates ESM-style exports to commonjs, but `components.js` exports its data like [this](https://github.com/PrismJS/prism/blob/3786f3965cb433cb16d9a2f467337916c28eb8db/components.js#L2).

Reply:
Yes this is correct. The following TypeScript:

```ts
import { core } frm 'prismjs/components';

console.log(core);
```

will be transpiled to:

```js
const prismjs_components_1 = require('prismjs/components');

console.log(prismjs_components_1.core);
```